js/src/common/components/SelectDropdown.js

```diff
@@ -12,6 +12,8 @@ import icon from '../helpers/icon';
 function isActive(vnode) {
   const tag = vnode.tag;
 
+  if (typeof tag === 'string') return false;
```

Comment:
I maintain my question of why? Not necessarily opposed to the change, but it needs a comment explaining it